introduction.md

```diff
@@ -1,2 +1,5 @@
 Good Evening. I am hcab. This is me introducing myself. Cheers!
-My favorite Code School path is javascript 
+My favorite Code School path is javascript
+* Git
+* JavaScript
+* AngularJS 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?